controllers/jobs.js

```diff
@@ -16,11 +25,31 @@ const createJob = async (req, res) => {
 }
 
 const updateJob = async (req, res) => {
-    res.send('update job');
+    const { body:{company, position}, user: {userId}, params: {id:jobId} } = req;
```

Comment:
I'd consider separating any large, multiple key destructing like this onto separate lines for easier reading:
```node.js
const {
  body: { company, position }, 
  user: { userId }, 
  params: { id: jobId }
} = req;
```